pkg/crypto/helpers_test.go

```diff
@@ -0,0 +1,35 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func TestTopicFromValidCert(t *testing.T) {
+	certificate, _ := ReadPEMCertificateFile("testdata/mock_push_cert.pem")
+	pushTopic, err := TopicFromCert(certificate)
+
+	if err != nil {
+		t.Errorf("fail %s", err)
+	}
+
+	if have, want := pushTopic, "com.apple.mgmt.External.18a16429-886b-41f1-9c30-2bd04ae4fc37"; have != want {
+		t.Errorf("have %d, want %d", have, want)
```

Comment:
%d is for integers. This should be a `%s`

Reply:
Doh.... Last minute refactoring with some copy/paste. :)